clouddq/main.py

```diff
@@ -276,15 +276,12 @@ def main(  # noqa: C901
             "deprecated in v1.0.0. Please migrate to use native-flags for "
             "specifying connection configs instead."
         )
-    if (
-        not dbt_profiles_dir
-        and (  # noqa: W503
-            not gcp_project_id or not gcp_bq_dataset_id or not gcp_region_id
-        )
-    ) or (dbt_profiles_dir and (gcp_project_id or gcp_bq_dataset_id or gcp_region_id)):
+    if (not dbt_profiles_dir and (not gcp_project_id or not gcp_bq_dataset_id)) or (
+        dbt_profiles_dir and (gcp_project_id or gcp_bq_dataset_id)
+    ):
         raise ValueError(
-            "CLI input must define connection configs using '--dbt_profiles_dir' or "
-            "using '--gcp_project_id', '--gcp_bq_dataset_id', and '--gcp_region_id'."
+            "CLI input must define connection configs using either '--dbt_profiles_dir' "
+            "or ('--gcp_project_id', '--gcp_bq_dataset_id', '--gcp_region_id')."
```

Comment:
I'm not sure the notation is going to be understood by everyone. Maybe put "...or one of: --gcp_project_id, ..."

Reply:
ack & fixed